src/Rule/RegexHandler.php

```diff
@@ -14,7 +14,7 @@
 /**
  * Validates that the value matches the pattern specified in constructor.
```

Comment:
I'd either mentioned possible inversion option, because current statement is not 100% correct or just provided the link to the handler.

Reply:
The link is there.